rement already satisfied: networkx in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch) (3.4.2)
Requirement already satisfied: jinja2 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch) (3.1.6)
Requirement already satisfied: fsspec in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch) (2025.3.2)
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/mkl/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/mkl/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/mkl/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/mkl/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/mkl/
Collecting mkl<=2021.4.0,>=2021.1.1 (from torch)
  Downloading http://repo.samsungds.net/artifactory/api/pypi/pypi-torch/mkl-2021.4.0-py2.py3-none-win_amd64.whl (228.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 228.5/228.5 MB 117.8 MB/s eta 0:00:00
Requirement already satisfied: numpy in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torchvision) (2.2.4)
Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torchvision) (11.1.0)
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/intel-openmp/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/intel-openmp/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/intel-openmp/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/intel-openmp/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/intel-openmp/
Collecting intel-openmp==2021.* (from mkl<=2021.4.0,>=2021.1.1->torch)
  Downloading http://repo.samsungds.net/artifactory/api/pypi/pypi-torch/intel_openmp-2021.4.0-py2.py3-none-win_amd64.whl (3.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.5/3.5 MB 105.0 MB/s eta 0:00:00
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/tbb/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/tbb/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/tbb/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/tbb/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/tbb/
Collecting tbb==2021.* (from mkl<=2021.4.0,>=2021.1.1->torch)
  Downloading http://repo.samsungds.net/artifactory/api/pypi/pypi-torch/tbb-2021.11.0-py3-none-win_amd64.whl (298 kB)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from jinja2->torch) (3.0.2)
Requirement already satisfied: mpmath<1.4,>=1.1.0 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from sympy->torch) (1.3.0)
Installing collected packages: tbb, intel-openmp, mkl, torch, torchvision, torchaudio
Successfully installed intel-openmp-2021.4.0 mkl-2021.4.0 tbb-2021.11.0 torch-2.3.1+cu121 torchaudio-2.3.1+cu121 torchvision-0.18.1+cu121

mgoog1.ko@KORCO095937 MINGW64 ~/Desktop/pcsllm/rag_project
$ pip uninstall torch torchvision torchaudio -y
Found existing installation: torch 2.3.1+cu121
Uninstalling torch-2.3.1+cu121:
  Successfully uninstalled torch-2.3.1+cu121
Found existing installation: torchvision 0.18.1+cu121
Uninstalling torchvision-0.18.1+cu121:
  Successfully uninstalled torchvision-0.18.1+cu121
Found existing installation: torchaudio 2.3.1+cu121
Uninstalling torchaudio-2.3.1+cu121:
  Successfully uninstalled torchaudio-2.3.1+cu121

mgoog1.ko@KORCO095937 MINGW64 ~/Desktop/pcsllm/rag_project
$ pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121
Looking in indexes: https://download.pytorch.org/whl/cu121, http://nexus.adpaas.cloud.samsungds.net/repository/dataservice-pypi/simple
Looking in links: http://repo.samsungds.net/artifactory/api/pypi/pypi-torch/torch_stable.html
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/torch/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/torch/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/torch/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, '현
재 연결은 원격 호스트에 의해 강제로 끊겼습니다', None, 10054, None))': /whl/cu121/torch/
ERROR: Operation cancelled by user

mgoog1.ko@KORCO095937 MINGW64 ~/Desktop/pcsllm/rag_project
$ cd torch/

mgoog1.ko@KORCO095937 MINGW64 ~/Desktop/pcsllm/rag_project/torch
$ pip install torch-2.1.0+cu121-cp310-cp310-win_amd64.whl
pip install torchvision-0.16.0+cu121-cp310-cp310-win_amd64.whl
pip install torchaudio-2.1.0+cu121-cp310-cp310-win_amd64.whl
Looking in indexes: http://repo.samsungds.net/artifactory/api/pypi/pypi-remote/simple, http://nexus.adpaas.cloud.samsungds.net/repository/dataservice-pypi/simple
Looking in links: http://repo.samsungds.net/artifactory/api/pypi/pypi-torch/torch_stable.html
Processing c:\users\mgoog1.ko\desktop\pcsllm\rag_project\torch\torch-2.1.0+cu121-cp310-cp310-win_amd64.whl
Requirement already satisfied: filelock in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (3.18.0)
Requirement already satisfied: typing-extensions in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (4.13.2)
Requirement already satisfied: sympy in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (1.13.1)
Requirement already satisfied: networkx in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (3.4.2)
Requirement already satisfied: jinja2 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (3.1.6)
Requirement already satisfied: fsspec in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from torch==2.1.0+cu121) (2025.3.2)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from jinja2->torch==2.1.0+cu121) (3.0.2)
Requirement already satisfied: mpmath<1.4,>=1.1.0 in c:\users\mgoog1.ko\appdata\local\programs\python\python310\lib\site-packages (from sympy->torch==2.1.0+cu121) (1.3.0)
Installing collected packages: torch
